GSA/digital-gov-drupal
# DIGITAL-154: Enable auto logout module

## Jira ticket




[DIGITAL-154](https://cm-jira.usa.gov/browse/DIGITAL-154)

## Purpose



This Pr will set up the auto logout module for all users to be logged out after 12 hours no matter the role or activity 

## Includes the following PRs that must be merged first



## Deployment and testing

### Local Setup



`lando rebuild -y`

### QA/Testing instructions



1. Visit the extend page and check that `auto logout` is enabled 
2. visit `admin/config/people/autologout` and update the timer to 60 secs and verify that you are logged out after the time as expired

## Checklist for the Developer




## Checklist for the Peer Reviewers
